underlay/src/main/resources/config/datamapping/sd/entity/cpt4/entity.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "cpt4",
+  "allInstancesSqlFile": "all.sql",
+  "attributes": [
+    { "name": "id", "dataType": "INT64" },
+    { "name": "concept_id", "dataType": "INT64" },
+    { "name": "name", "dataType": "STRING" },
+    { "name": "vocabulary", "dataType": "STRING", "valueFieldName": "type" },
+    { "name": "is_standard", "dataType": "STRING", "isComputeDisplayHint": true },
+    { "name": "concept_code", "dataType": "STRING" },
```

Comment:
Is the only change here renaming the attribute from `code` -> `concept_code`? If so, could you just modify the existing entity in place? SD is the only underlay that uses the `omop/cpt4` definition, aside from the test underlays, so you don't need to worry about breaking other underlays in this case.